package/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM alpine:3.13
```

Comment:
Can you use `registry.access.redhat.com/ubi7/ubi-minimal:latest` and add `RUN microdnf update -y && \ 
    rm -rf /var/cache/yum`? That is the image we are trying to use more consistently as it is approved by certain orgs and the command should ensure the packages are up to date.

Reply:
Yes, this will require a little bit more testing.

Reply:
I had to use `ubi8` for arm64 support.